orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -342,6 +358,16 @@ def _anonymise_study_instances(
             logger.debug("Reading file {}", file)
             dataset = dcmread(file)
 
+            if series_to_keep and dataset.SeriesInstanceUID not in series_to_keep:
+                logger.debug(
+                    "Skipping series {} for study {} as series not in series_to_keep",
+                    dataset.SeriesInstanceUID,
+                    study_info,
+                )
+                key = "DICOM instance discarded as series not requested"
+                skipped_instance_counts[key] += 1
+                continue
```

Comment:
this seemed easier than handling the case where `payload["ResourceIDs"]` is either a list of Study resources or list of Series resources, but I never tried to do it so I may be wrong? I also assumed it would be pretty rare that we would have series in Orthanc Raw that we don't want to import to Anon?